doc/releases/changelog-0.23.0.md

```diff
@@ -444,6 +444,9 @@
   accessed via the top-level `qml` namespace.
   [(#2396)](https://github.com/PennyLaneAI/pennylane/pull/2396)
 
+* Raise a warning where caching produces identical shot noise on execution results with finite shots.
```

Comment:
Note: this would go in the section below (Deprecations)

Reply:
What is being deprecated?

Reply:
No worries :D